core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -1224,6 +1224,10 @@ public int getQueryConcurrentRunningThresholdForProject() {
         return Integer.parseInt(getOptional("kylin.query.project-concurrent-running-threshold", "0"));
     }
 
+    public String[] getHighPriorityProjects() {
```

Comment:
what is this for?

Reply:
This part has been refactored. It's useless and I'll remove it.